pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -227,89 +205,11 @@ func enablePredicate(args framework.Arguments) predicateEnable {
 }
 
 func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
-	pl := ssn.PodLister
 	nodeMap := ssn.NodeMap
 
 	pCache := predicateCacheNew()
 	predicate := enablePredicate(pp.pluginArguments)
 
-	// Register event handlers to update task info in PodLister & nodeMap
-	ssn.AddEventHandler(&framework.EventHandler{
-		AllocateFunc: func(event *framework.Event) {
```

Comment:
This should not be all removed, just remove device related codes, because other parts are needed by predict plugin.